groclient/lib.py

```diff
@@ -6,10 +6,14 @@
 """
 
 from builtins import str
-from math import ceil
 from groclient import cfg
-from groclient.constants import REGION_LEVELS
-from groclient.utils import dict_reformat_keys, str_snake_to_camel, str_camel_to_snake, list_chunk
+from collections import OrderedDict
+from groclient.constants import REGION_LEVELS, DATA_SERIES_UNIQUE_TYPES_ID
+from groclient.utils import (dict_reformat_keys,
```

Comment:
The document says not to import methods directly
https://grointelligence.atlassian.net/wiki/spaces/EN/pages/17530900/Python+PEP+8+Standards+and+Gro+Preferences

Reply:
Updated!